tools/diagnostics/nsdiff

```diff
@@ -0,0 +1,875 @@
+#!/usr/bin/env python
+
+# README:
+#
+# This script assumes:
+# --------------------
+#
+# 1. The entirety of a node's state is bundled as a zip or tar file
+# 2. The zip/tar file has the name <node_name>.zip or <node_name>.tar.gz where
+#    <node_name> is the name of the node.
+# 3. Both zip/tar files to be compared are on the filesystem where nsdiff is
+#    installed. In other words, nsdiff does not attempt to connect or collect
+#    state data from remote nodes.
+# 4. nscapture is/was used to capture state passed as state1 and state2
+#
+# Installation:
+# -------------
+#
+# If this script is used on a server or workstation that has indy-node installed
+# and indy-node is known to be working, you do not need to install anything.
+# Otherwise, the following information will be helpful.
+#
+# This script depends on leveldb/rocksdb python bindings which in turn rely on
+# the leveldb C runtime libs and headers. You must install leveldb/rocksdb
+# dependencies. See Installation > Prerequisites section below.
```

Comment:
These tools are not intended as solely as development tools. They are stand-alone tools. Their requirement should be equal or less than to running the node. 

So they may be duplication since they would be installed with the node.